emthree/emthree.lua

```diff
@@ -923,6 +923,8 @@ function M.on_input(board, action)
 	local block = M.get_block(board, x, y)
 	if block and block.blocker then
 		return
+    elseif block and block.spawner then
```

Comment:
Could you please fix the indentation?

We could combine the two checks:

```
if block and (block.blocker or block.spawner) then
	return
end
```

Reply:
Done, thanks! Editing it in `vim` wasn't the best idea :smile: